backend/bitboxbase/rpcmessages/rpcmessages.go

```diff
@@ -14,13 +14,15 @@ const (
 	OpUCanHasVerificationProgress = "v"
 	// OpServiceInfoChanged notifies when the GetServiceInfo data changed.
 	OpServiceInfoChanged = "s"
+	// OpBaseUpdateProgressChanged notifies when the BaseUpdateProgress changes while performing a Base Update.
+	OpBaseUpdateProgressChanged = "u"
 )
 
 /*
 Put Incoming Args below this line. They should have the format of 'RPC Method Name' + 'Args'.
 */
 
-// UserAuthenticateArgs is a struct that holds the arguments for the UserAuthenticate RPC call. The first rpc call should always authenticate first
+// UserAuthenticateArgs is an struct that holds the arguments for the UserAuthenticate RPC call
```

Comment:
Why change "a struct" to "an struct"? It should be the former

Reply:
Must have happend while coping from the `base/rpcmessages.go`.